crates/core/tedge_api/src/alarm.rs

```diff
@@ -49,6 +49,9 @@ pub enum ThinEdgeJsonDeserializerError {
         error: String,
         payload: String,
     },
+
+    #[error("Parsing of alarm message to string received on topic: {topic} failed due to error: {error}")]
+    FailedToParsePayloadToString { topic: String, error: String },
```

Comment:
This error message is unclear for a non rust developer.

```suggestion
    #[error("Failed to parse as an UTF-8 string the payload received on topic: {topic}, due to error: {error}.")]
    FailedToParsePayloadToString { topic: String, error: String },
```

Reply:
updated the message